Makefile

```diff
@@ -6,23 +6,22 @@ GOLIST=go list
 GOFMT=gofmt
 BUILD_GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 
-IMAGE_BUILDER_OPTS=
-IMAGE_BUILDER?=imagebuilder
-IMAGE_BUILD=$(IMAGE_BUILDER)
-export IMAGE_TAG_CMD?=docker tag
+IMAGE_BUILDER?=@docker
+IMAGE_BUILD_OPTS?=
+DOCKERFILE?=Dockerfile
 
 export APP_NAME=sriov-network-operator
 APP_REPO=github.com/openshift/$(APP_NAME)
 TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
-export IMAGE_TAG=quay.io/pliurh/$(APP_NAME):latest
+IMAGE_TAG?=quay.io/openshift/$(APP_NAME):latest
```

Comment:
OpenShift uses `Dockerfile.rhel7` for image building, we might want to name this tag as a community version.
For example, to make this consistent with other sriov components like device plugin or cni, use `nfvpe` prefix and setup travis to upload image upon any repo changes.

Reply:
+1, or we can just use a environment variable here, so that developers can chose whatever repo name they want to use.

Reply:
I am working on another PR for travis job
@zshi-redhat @pliurh do you have other comments?